src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts

```diff
@@ -307,3 +331,13 @@ export class OpenSheetMusicDisplay {
     }
     //#endregion
 }
+
+export interface OSMDOptions {
+    autoBeam?: boolean; // not yet supported. will always autoBeam.
+    autoResize?: boolean; // default is false
+    autoStem?: boolean; // not yet supported. will always autoStem
+    backend?: string; // Backend will be SVG if backend === "svg", otherwise Canvas
+    disableCursor?: boolean; // will override this part of drawingParameters
+    drawingParametersEnum?: DrawingParametersEnum;
```

Comment:
I don't really like this, what's the reason not to use something like:

```
drawingParameters?: DrawingParametersEnum | string;
```

Reply:
It's because we need to handle a string differently than an Enum,
and [type checks in Typescript type still aren't ideal](https://stackoverflow.com/questions/12789231/class-type-check-with-typescript/12789324#12789324).
But since we use `instanceof` a lot in OSMD, i guess we can do it here, too.

Reply:
Also, it seems like instanceof doesn't work with Enums, or something.

![image](https://user-images.githubusercontent.com/33069673/45489969-2760c280-b766-11e8-9cc7-eb275e55d892.png)

<details>
<summary>Details</summary>

```typescript
[ts] The left-hand side of an 'instanceof' expression must be of type 'any', an object type or a type parameter.
(property) IOSMDOptions.drawingParameters?: string | DrawingParametersEnum.Default | DrawingParametersEnum.Leadsheet | DrawingParametersEnum.Preview | DrawingParametersEnum.Thumbnail
```

```typescript
        this.drawingParameters = new DrawingParameters();
        if (options.drawingParameters) {
            if (options.drawingParameters instanceof DrawingParametersEnum) {
                this.drawingParameters.DrawingParametersEnum = options.drawingParametersEnum;
            } else if (options.drawingParameters instanceof string) {
                this.drawingParameters.DrawingParametersEnum =
                    DrawingParameters.DrawingParametersStringToEnum(options.drawingParametersString);
            }
        } else {
            this.drawingParameters.DrawingParametersEnum = DrawingParametersEnum.Default;
        }
```

</details>

Reply:
You don't need to do the typecheck if you use string-based enums.
https://goo.gl/HUdLTh